openff/toolkit/tests/test_molecule.py

```diff
@@ -2492,6 +2491,8 @@ def test_equality(self):
 
     def test_add_conformers(self):
         """Test addition of conformers to a molecule"""
+        from openmm import unit as openmm_unit
+
```

Comment:
👍 

At some point in the future we'll need to figure out how to handle test skips for openmm-free builds. But I don't think we need to handle that in this PR.

Reply:
We should be able to get away with liberal use of [`skip_if_missing`](https://github.com/openforcefield/openff-utilities/blob/6db2421dec43f9e6e34ca38fa39f9bac86e00c85/openff/utilities/testing.py#L6) but I won't worry about that now